metadata-ingestion/tests/unit/test_codegen.py

```diff
@@ -67,3 +75,74 @@ def test_urn_annotation():
     assert FineGrainedLineageClass.RECORD_SCHEMA.fields_dict["upstreams"].get_prop(
         "urn_is_array"
     )
+
+
+def _add_to_registry(entity: str, aspect: str) -> None:
+    import pathlib
+
+    from ruamel.yaml import YAML
+
+    registry = pathlib.Path("../metadata-models/src/main/resources/entity-registry.yml")
+
+    yaml = YAML()
+
+    doc = yaml.load(registry)
+
+    for entry in doc["entities"]:
+        if entry["name"] == entity:
+            entry["aspects"].append(aspect)
+            break
+    else:
+        raise ValueError(f'could not find entity "{entity}" in registry')
```

Comment:
As a touch of newbie proofing, could we add a note here about where exactly that registry lives? 

Is it just ../metadata-models/src/main/resources/entity-registry.yml?